### PR TITLE
refactor(relayer): Assume only v3 deposits

### PR DESCRIPTION
**Pull request by pxrl**
Since the v3 launch is now > 1 day old, no operational relayers should be seeing new v2 events. This commit starts the process of stripping out v2 support.

### CODE REVIEW

File: src/relayer/Relayer.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ export class Relayer {
       }
 
       // Skip deposits with quoteTimestamp in the future (impossible to know HubPool utilization => LP fee cannot be computed).
-      if (quoteTimestamp > hubPoolClient.currentTime) {
+      if (deposit.quoteTimestamp > hubPoolClient.currentTime) {
```
Comment (nicholaspai) [inline src/relayer/Relayer.ts:87]: Why this change?

**Review by nicholaspai — APPROVED** [review]

File: src/relayer/Relayer.ts (hunk 1 above)
Comment (pxrl) [reply src/relayer/Relayer.ts:87]: I exchanged it for `inputToken` and `outputToken` up on line 60. If we extract them all then it overflows to > 1 line and becomes one line per variable.
